wandb/sdk/internal/internal_api.py

```diff
@@ -70,6 +70,33 @@ class CreateArtifactFileSpecInput(TypedDict):
         mimetype: Optional[str]
         artifactManifestID: Optional[str]
 
+    class CreateArtifactResponseLatestArtifactInfo(TypedDict):
+        id: str
+        versionIndex: int
```

Comment:
(I'd like to define this as a subclass on `CreateArtifactResponse`, so the name could just be `LatestArtifactInfo`, but TypedDict rejects any declarations that aren't type annotations.)